### ISSUE TITLE
LibreOffice newer versions stability issues

### ISSUE THREAD

**Issue by gulien** [gotenberg/gotenberg#576]
I did a bit of research and came to the following result with a simple binary search:

Working | Not Working | Release Date
-- | -- | --
  | 7.7.1 (4.4.2.3) | 03.12.2022
  | 7.6.1 (7.4.1.2 40(Build:2)) | 03.10.2022
7.6.0 (7.3.5.2 30(Build:2)) |   | 01.09.2022
7.5.4 (7.3.5.2) |   |  
7.5.3 (7.3.4.1) |   | 07.06.2022

The version in brackets represents the used LibreOffice version in each gotenberg version.
That might explain the sudden issue on our end, as we used `gotenberg/gotenberg:7` as the image. I've created this issue on the 04th of October, and the update with the broken version of LO was released on the 03rd of October. Our CI/CD pipeline most certainly did a `docker-compose pull` which updated to the named version and explains the sudden error(s).

So, basically - the culprit is on of 1.185 commits between v7.4.1.2 and v7.3.5.2 (https://github.com/LibreOffice/core/compare/libreoffice-7.4.1.2...libreoffice-7.3.5.2) - fun to debug tbh

_Originally posted by @PierreSchwang in https://github.com/gotenberg/gotenberg/issues/516#issuecomment-1372607904_

**Comment by gulien** [comment]
Thanks @PierreSchwang for the investigation 👍

**Comment by gulien** [comment]
https://github.com/gotenberg/gotenberg/issues/442#issuecomment-1384186104

Gotenberg garbage collector causing an issue with newer LibreOffice versions?

**Comment by gulien** [comment]
https://github.com/gotenberg/gotenberg/issues/577#issuecomment-1401636345

**Comment by yan-hic** [comment]
> [#577 (comment)](https://github.com/gotenberg/gotenberg/issues/577#issuecomment-1401636345)
> 
> Lock can take longer than 10 seconds?

Faced that exact same error re lock timeout right off the bat when starting a private `gotenberg:7`.

@gulien would you advise to downgrade 7.4.2 until fixed ? What major features would one miss ?
Our use case: converting xlsx in landscape pdf

**Comment by gulien** [comment]
Hello @yan-hic,

You'll miss newer PDF Formats (PDF/A-2b & PDF/A-3b) but that's it 😉

**Comment by yan-hic** [comment]
@gulien would you consider a build using the next major release of libreoffice (7.5.2 can be downloaded) ? Wondering if stability is better (+ performance and allow new enhancements).

**Comment by gulien** [comment]
@yan-hic Indeed, might be interesting to test it when available on Debian bullseye.

**Comment by gulien** [comment]
Hello @yan-hic

Not yet. But it's on my TODO list!

**Comment by yan-hic** [comment]
> Not yet. But it's on my TODO list!

Just checking in as I was hoping to move out of 7.4.2. More recent `gotenberg` releases have not helped for our use case (mainly `xlsx` conversion), and 7.4.2 remains the most stable one - again, in our use case.

**Comment by gulien** [comment]
Hello @yan-hic,

I'm currently working on it, but, again, not ETA alas. Good news though: it seems that there are no more stability issues based on my local testing.

**Comment by novelnet** [comment]
@gulien Thanks!
